install/kubernetes/istio-one-namespace.yaml

```diff
@@ -270,7 +272,29 @@ spec:
           - --logtostderr
           - -v
           - "2"
+      - name: istio-proxy
+        image: gcr.io/istio-testing/proxy_debug:6f20fc31474dae9a944d47a364ded3bb3439c477
+        imagePullPolicy: IfNotPresent
+        ports:
+        - containerPort: 15004
+        args:
+        - proxy
+        - mixer
+        - -v
+        - "2"
+        - --controlPlaneAuthPolicy
+        - NONE #--controlPlaneAuthPolicy
```

Comment:
If you are not going to be using control plane authentication in the no auth mode, why add a sidecar ? We don’t need it at all right?

Reply:
Good point, and I think this will make the turning on/off experience smoother, just like we always have a CA up running.

Reply:
The ports that envoy listens on is different than what pilot or mixer listens on, also as part of istio-on-istio we want to get stats, check, report etc. It is not part of this PR but it allows for us to take advantage of istio features in future.